src/aig-common/modules/standard/components/ipp-modality/ipp-modality-form/ipp-modality-form.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
review this file completly. take inspiration from this:
[https://github.com/EneaSys/aig_console/blob/develop/src/aig-common/modules/generic/components/eopoo-type-new-update-form/eopoo-type-new-update-form.component.ts](url)

Reply:
This component also was modified as *-new-update-form component.